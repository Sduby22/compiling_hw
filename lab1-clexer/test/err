123.23
123.
0x123.213
0b00110
0b123
0x23213g
12321abc
"unterminated
'u
"newline with backslash\
newline"
@@@
/* unfinished block comment
