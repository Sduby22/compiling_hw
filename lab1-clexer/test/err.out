SYMTYPE	SYMATTR	SYMVALUE
T_NUM	ATTR_NUM_FLOAT	123.23
T_NUM	ATTR_NUM_FLOAT	123.
line 3: ERR_UNEXPECTED_IDENTIFIER 0x123.213
T_NUM	ATTR_NUM_BIN	0b00110
line 5: ERR_UNEXPECTED_IDENTIFIER 0b123
line 6: ERR_UNEXPECTED_IDENTIFIER 0x23213g
line 7: ERR_UNEXPECTED_IDENTIFIER 12321abc
line 9: ERR_UNTERMINATED_STRING_LITERAL "unterminated

line 9: ERR_UNTERMINATED_CHAR_LITERAL 'u
T_LITERAL	---	"newline with backslash\
newline"
line 12: ERR_UNEXPECTED_CHARACTER @
line 12: ERR_UNEXPECTED_CHARACTER @
line 12: ERR_UNEXPECTED_CHARACTER @
line 14: ERR_UNTERMINATED_BLOCK_COMMENT 
代码行数 : 13
总字节数 : 137
注释数 : 0
关键字数 : 0
标点符号数 : 0
数字常量数 : 3
字符串常量数 : 1
标识符数 : 0
未识别符号数 : 3
